sapper/src/routes/om-oss/[slug].svelte

```diff
@@ -0,0 +1,55 @@
+<script context="module">
+  import { API } from '../../services/api.js'
+
+  export async function preload({ params: { slug } }, session) {
+    return API({ fetch: this.fetch, session }).Coworkers.BySlug(slug)
+  }
+</script>
+
+<script>
+  import Animation from '../../components/animation/'
+  import Layout from '../../components/layout'
+  import Typography from '../../components/typography'
+  import md5 from 'md5'
+
+  export let coworker
+  const [workTitle, phone, email] = coworker.custom_excerpt.split('\n')
+</script>
+
```

Comment:
Missing meta-tag. For instance `<Meta title="Iteam | Om oss - {coworker.title}" />`